backend/prisma/migrations/20230205202717_create_user_table/migration.sql

```diff
@@ -0,0 +1,14 @@
+-- CreateTable
+CREATE TABLE `User` (
+    `id` INTEGER NOT NULL AUTO_INCREMENT,
+    `email` VARCHAR(191) NOT NULL,
+    `firstName` VARCHAR(191) NOT NULL,
+    `lastName` VARCHAR(191) NOT NULL,
+    `birthdate` DATETIME(3) NOT NULL,
+
+    UNIQUE INDEX `User_email_key`(`email`),
+    UNIQUE INDEX `User_firstName_key`(`firstName`),
+    UNIQUE INDEX `User_lastName_key`(`lastName`),
+    UNIQUE INDEX `User_birthdate_key`(`birthdate`),
```

Comment:
I don't think we're going to want to include a `UNIQUE INDEX` for these three fields. 

Otherwise there we won't be able to accommodate having two Kevins, or Two Justins

Reply:
yeah you don't have to update this file, just update the schema.prisma file and then redo the prisma generate command